back/app/policies/idea_comment_policy.rb

```diff
@@ -55,4 +53,14 @@ def check_commenting_allowed(comment, user)
     pcs = ParticipationPermissionsService.new
     !pcs.commenting_disabled_reason_for_idea comment.post, user
   end
+
+  def active_moderator?
+    return false unless active? && (admin? || user&.project_moderator?)
+
+    # Needed to permit project moderator to export all comments,
+    # without processing :idea_comment symbols for idea_comments from unmoderated projects.
+    return true if record.instance_of?(Symbol)
```

Comment:
I have the same misgivings about this approach as described for the similar approach used in the Idea policy, described [here](https://github.com/CitizenLabDotCo/citizenlab/pull/7635/files#r1569009998).